sqlx-macros-core error build with `--cfg sqlx_macros_unstable`

### Bug Description
Clean build of project with sqlx and migrate feature is not working when cfg `sqlx_macros_unstable` is present.
erorr:
```sh
env RUSTFLAGS='--cfg sqlx_macros_unstable' cargo build
   Compiling sqlx-macros-core v0.7.0-alpha.2
error[E0433]: failed to resolve: use of undeclared crate or module `tracked_path`
   --> /home/p/.cargo/registry/src/index.crates.io-6f17d22bba15001f/sqlx-macros-core-0.7.0-alpha.2/src/migrate.rs:142:9
    |
142 |         tracked_path::path(path);
    |         ^^^^^^^^^^^^ use of undeclared crate or module `tracked_path`
```
### Minimal Reproduction
```sh
$ cargo new foobar && cd foobar;
$ cargo add sqlx@0.7.0-alpha.2 --features migrate
$ env RUSTFLAGS='--cfg sqlx_macros_unstable' cargo build
```
### Info
* SQLx version: 0.7.0-alpha.2
* SQLx features enabled: migrate
* Database server and version: --
* Operating system: Linux version 6.2.2-arch1-1 (linux@archlinux)
* `rustc --version`: rustc 1.70.0-nightly (13afbdaa0 2023-03-17)

--- a/sqlx-macros-core/src/lib.rs
+++ b/sqlx-macros-core/src/lib.rs
@@ -14,6 +14,11 @@
 //! And then make releases in lockstep with `sqlx-core`. We recommend all driver crates, in-tree
 //! or otherwise, use the same version numbers as `sqlx-core` to avoid confusion.
 
+#![cfg_attr(
+    any(sqlx_macros_unstable, procmacro2_semver_exempt),
+    feature(track_path)
+)]
+
 use once_cell::sync::Lazy;
 
 use crate::query::QueryDriver;

--- a/sqlx-macros-core/src/migrate.rs
+++ b/sqlx-macros-core/src/migrate.rs
@@ -1,3 +1,6 @@
+#[cfg(any(sqlx_macros_unstable, procmacro2_semver_exempt))]
+extern crate proc_macro;
+
 use proc_macro2::TokenStream;
 use quote::{quote, ToTokens, TokenStreamExt};
 use sha2::{Digest, Sha384};

